pyproject.toml

```diff
@@ -7,42 +7,34 @@ max-branches=50
 max-nested-blocks=10
 max-statements=200
 
-[tool.isort]
-profile = "black"
-src_paths = ["src", "tests/ci", "tests/sqllogic"]
-
 [tool.pylint.FORMAT]
 #ignore-long-lines = (# )?<?https?://\S+>?$
 
 [tool.pylint.'MESSAGES CONTROL']
           # pytest.mark.parametrize is not callable (not-callable)
 disable = '''
-          pointless-string-statement,
-          line-too-long,
```

Comment:
this is excessive check for ci scripts

Reply:
It's opinionated. We have 200 lines limit, and it's not nice to read long lines. Many of places where it complained, became better

[one](https://github.com/ClickHouse/ClickHouse/pull/70013/files#diff-b3f5f3ea620ee8e310f6bdfab21608735e58c4d04402cd5a70fd267047b36a1aR72)
[two](https://github.com/ClickHouse/ClickHouse/pull/70013/files#diff-b3f5f3ea620ee8e310f6bdfab21608735e58c4d04402cd5a70fd267047b36a1aR466-R469)
[three](https://github.com/ClickHouse/ClickHouse/pull/70013/files#diff-4a5684a1ef65d804c900f98ec655040f7e5bfff1b446b2509364fb9419a2342fR159-R167)

In any case, if you disagree, you can enable it locally in the file. But have it globally enabled for the whole repo is the idea I disagree